Content.Shared/Morgue/SharedMorgue.cs

```diff
@@ -3,14 +3,22 @@
 namespace Content.Shared.Morgue;
 
 [Serializable, NetSerializable]
-public enum MorgueVisuals
+public enum MorgueVisuals : byte
 {
+    Contents
+}
+
+[Serializable, NetSerializable]
+public enum MorgueContents : byte
+{
+    Empty,
     HasMob,
     HasSoul,
+    HasContents,
```

Comment:
on `HasContents`, IIRC when reviewing the ECS PR I initially said this should just be a generic entity storage thing (and it is, see `StorageVisuals.HasContents`), but just because of how the generic visualizer works, these different values that map onto different layer states need to be part of the same enum. So there's a bit of overlap with the generic storage enum